pyrightconfig.json

```diff
@@ -5,5 +5,6 @@
         "htmltools/",
         "examples/",
     ],
-    "reportImportCycles": false
+    "reportImportCycles": false,
+    "reportPrivateUsage": "none",
```

Comment:
This setting seems not to have an effect? (The `type: ignore` is still needed in _core.py even with this setting.) Probably because we set the mode to `basic` in general, and then override it with `strict` for specific directories. So I think this change can be removed.